app/controllers/admin/residents_controller.rb

```diff
@@ -29,6 +29,20 @@ def show
       @resident = @house.residents.find(params[:id])
     end
 
+    def edit
+      @resident = @house.residents.find(params[:id])
```

Comment:
What do you think about adding a private method to set the `@resident` as you did for the `@house`? In this way, you could avoid repeating this line in L29 and L37  😉